packages/components/socialFeed/NewsFeed/NewsFeedQueries.ts

```diff
@@ -256,3 +258,12 @@ export const generatePostMetadata = ({
     updatedAt: new Date().toISOString(),
   };
 };
+
+export const isValidMusicAudioPost = (post: Post) =>
+  post.category === PostCategory.MusicAudio &&
+  ZodTrack.safeParse(JSON.parse(post.metadata)).success;
```

Comment:
this function can throw if JSON.parse fails, maybe wrap it in a try catch

Reply:
Oki, done